### PR TITLE
rst2html5: 1.9.4 -> 1.10.6

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Motivation for this change

Now with wheels!

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/text/rst2html5/default.nix [hunk 1]
```diff
@@ -1,19 +1,22 @@
-{ stdenv, fetchurl, pythonPackages }:
+{ lib, python3Packages }:
 
-pythonPackages.buildPythonPackage rec {
+let
   pname = "rst2html5";
-  version = "1.9.4";
+  version = "1.10.6";
```
Comment (SuperSandro2000) [inline pkgs/tools/text/rst2html5/default.nix:5]: Why are you always using let in? rec is 2 lines shorter and has less repetition. This makes only sense to use when you input another derivation, an icon or something similar.

**Comment by SuperSandro2000** [comment]
Does not build for me:

```

trying https://files.pythonhosted.org/packages/source/r/rst2html5/rst2html5-1.10.6.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404

trying https://pypi.io/packages/source/r/rst2html5/rst2html5-1.10.6.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   122  100   122    0     0   1297      0 --:--:-- --:--:-- --:--:--  1297
100   279  100   279    0     0    948      0 --:--:-- --:--:-- --:--:--   948
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
error: cannot download rst2html5-1.10.6.tar.gz from any mirror
builder for '/nix/store/zsh16gcmqx6wmlgyha4gjvfywn81w7v2-rst2html5-1.10.6.tar.gz.drv' failed with exit code 1
cannot build derivation '/nix/store/kw9m16ip9f0asdw6qhj9b8wm78pf83ym-python3.8-rst2html5-1.10.6.drv': 1 dependencies couldn't be built
error: build of '/nix/store/kw9m16ip9f0asdw6qhj9b8wm78pf83ym-python3.8-rst2html5-1.10.6.drv' failed
```

@AndersonTorres Did the version get yanked?

File: pkgs/tools/text/rst2html5/default.nix (hunk 1 above)
Comment (AndersonTorres) [reply pkgs/tools/text/rst2html5/default.nix:5]: `rec` scrambles the scope of its variables, and I don't think it is bad to "factor" a variable like `pname`.

**Comment by bobrik** [comment]
The filename is different. Compare 1.9.4:

* https://pypi.org/project/rst2html5/1.9.4/#files

image

To 1.10.6:

* https://pypi.org/project/rst2html5/1.10.6/#files

image

**Pull request merged**